.evergreen/config_generator/components/earthly.py

```diff
@@ -38,7 +38,7 @@
 "Valid options for the SASL configuration parameter"
 TLSOption = Literal["OpenSSL", "off"]
 "Options for the TLS backend configuration parameter (AKA 'ENABLE_SSL')"
-CxxVersion = Literal["none"] # TODO: Once CXX-3103 is released, add latest C++ release tag.
+CxxVersion = Literal["r4.1.0", "none"]
```

Comment:
Is `4.1.0` the only version we want to test at the moment? Do we care to test `master`?

Reply:
I think testing `master` seems reasonable (and was [done before](https://github.com/mongodb/mongo-c-driver/blob/86037268ccaab73a841219225d49a0c32c5024c4/.evergreen/config_generator/components/earthly.py#L31)).